Documentation/source/Customization guide.md

```diff
@@ -24,26 +24,26 @@ Some background and text colors use the `GiniColor` type with which you can set
 </br>
 
 ##### 1. Background color
-- Background color &#8594; `GiniPayBusinessConfiguration.paymentScreenBackgroundColor` using `GiniColor` with dark mode and light mode colors
+- Background color &#8594; `GiniHealthConfiguration.paymentScreenBackgroundColor` using `GiniColor` with dark mode and light mode colors
```

Comment:
I thought we agreed that the source code won't be renamed. This includes code snippets in the docs. We'll do the source code renaming and update the snippets in the docs at a later date.

I'm fine with it now that you already did the work 🙂. The Health SDK docs are anyhow just for show until we rename the source code and make the first public release.

Reply:
overdone.